package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mavenlink/design-system",
-  "version": "0.7.0",
+  "version": "0.7.0-buttons-1",
```

Comment:
Looks like we didn't need a pre-release version. The custom forms repository can pull in a github dependency if we use a non-SSH git protocol during development.